Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -256,20 +236,23 @@ public async Task InstallPackage(InstallationInfo package, CancellationToken can
 
             var linkedToken = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, innerCancellationTokenSource.Token).Token;
 
-            PackageInstalling?.Invoke(this, package);
+            await _eventManager.PublishAsync(new PluginInstallingEventArgs(package)).ConfigureAwait(false);
 
             try
             {
-                await InstallPackageInternal(package, linkedToken).ConfigureAwait(false);
+                var isUpdate = await InstallPackageInternal(package, linkedToken).ConfigureAwait(false);
```

Comment:
It's not very clear what the boolean means just by looking at the method signature...

Reply:
Would a comment suffice, or are you thinking more detail? Holding on merge until question is answered.